Source repository link added in the Plug-in Selection Spy (2)

amelodev opened the pull request:
This request replace PR #2000, fixing an incorrect commit name.

HannesWell commented on ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/util/ManifestUtils.java:
```diff
@@ -393,4 +396,40 @@ public static void parseRequiredEEsFromFilter(String eeFilter, Consumer<String>
+						ICoreConstants.ECLIPSE_SOURCE_REFERENCES + " not found for bundle : " + bundle);
+			}
+		}
+		return strReturn;
```
```suggestion
		return null;
```

iloveeclipse commented:
@amelodev : merge commits are not allowed. Please always rebase your branch on top of master, do not merge!

amelodev commented:
> @amelodev : merge commits are not allowed. Please always rebase your branch on top of master, do not merge!

Sorry, i updated PR locally and removed the ugly merge commit :-)

opcoach commented:
Thank you @amelodev and @HannesWell
